plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -347,40 +277,40 @@ func (a *traceflowPlugin) traceflowHandler(request service.Request) (component.C
 	card.AddAction(genGraph)
 
 	graphCard := component.NewCard(component.TitleFromString("Antrea Traceflow Graph"))
-	if a.lastTfName != "" {
+	if lastTfName != "" {
 		// Invoke GenGraph to show
 		log.Printf("Generating content from CRD...")
 		ctx := context.Background()
-		tf, err := a.client.OpsV1alpha1().Traceflows().Get(ctx, a.lastTfName, metav1.GetOptions{})
+		tf, err := client.OpsV1alpha1().Traceflows().Get(ctx, lastTfName, metav1.GetOptions{})
 		if err != nil {
-			log.Printf("Failed to generate content from CRD, lastTfName %v, err: %s", a.lastTfName, err)
-			return component.ContentResponse{}, nil
+			log.Printf("Failed to generate content from CRD, lastTfName %v, err: %s", lastTfName, err)
+			return component.EmptyContentResponse, nil
 		}
 		log.Printf("Traceflow Results: %+v", tf)
-		a.graph = graphviz.GenGraph(tf)
-		log.Printf("Generated content from CRD successfully, lastTfName %v", a.lastTfName)
+		graph = graphviz.GenGraph(tf)
+		log.Printf("Generated content from CRD successfully, lastTfName %v", lastTfName)
 	}
-	if a.graph != "" {
-		graphCard.SetBody(component.NewGraphviz(a.graph))
+	if graph != "" {
+		graphCard.SetBody(component.NewGraphviz(graph))
 	} else {
 		graphCard.SetBody(component.NewText(""))
 	}
 	listSection := layout.AddSection()
 	err := listSection.Add(card, component.WidthFull)
 	if err != nil {
 		log.Printf("Failed to add card to section: %s", err)
-		return component.ContentResponse{}, nil
+		return component.EmptyContentResponse, nil
 	}
-	if a.graph != "" {
+	if graph != "" {
 		err = listSection.Add(graphCard, component.WidthFull)
 		if err != nil {
 			log.Printf("Failed to add graphCard to section: %s", err)
-			return component.ContentResponse{}, nil
+			return component.EmptyContentResponse, nil
 		}
 	}
 
-	tfCols := component.NewTableCols(tfNameCol, srcNamespaceCol, srcPodCol, dstNamespaceCol, dstPodCol, crdCol, phaseCol, ageCol)
-	tfTable := component.NewTableWithRows("Trace List", "", tfCols, a.getTfRows())
+	tfCols := component.NewTableCols(tfNameCol, srcNamespaceCol, srcPodCol, dstNamespaceCol, dstPodCol, traceflowCrdCol, phaseCol, ageCol)
+	tfTable := component.NewTableWithRows("Trace List", "", tfCols, getTfRows())
```

Comment:
should it be `traceflowTitle` instead of `"Trace List"` here? I don't understand where `"Trace List"` comes from

Reply:
+1